07_errors/sirigithub/error.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import "fmt"
+
+type Error struct {
+	Message string
+	Code    int
+}
```

Comment:
Since you will often wrap other errors, I would include an `err error` field here too so you can store the error returned from leveldb.